onnx/utils.py

```diff
@@ -18,6 +17,5 @@ def polish_model(model):  # type: (ModelProto) -> ModelProto
     onnx.checker.check_model(model)
     onnx.helper.strip_doc_string(model)
     model = onnx.shape_inference.infer_shapes(model)
-    model = onnx.optimizer.optimize(model)
```

Comment:
Given that optimizer has been removed here, do we still want to keep this API `polish_model`? All it is doing is running the checker and shape inference. strip_doc_string was there mainly for models coming from PyTorch because they had long and unwieldy doc strings. That has  been fixed. In fact, I would say removing doc strings is possibly not a good thing. Given all this, I think this API is now just an ad-hoc collection of steps with no real motivating use case anymore.
Therefore, we should consider removing it as part of this PR.

Reply:
Agree. I think they are for the same purpose currently.